modules/test/playwright/tests/commerce/commerce-product-content-search-web/sort.spec.ts

```diff
@@ -0,0 +1,197 @@
+	await commerceThemeMiniumCatalogPage.configurationIFrameDefaultSortingDropdownMenu.selectOption(
+		sortingOption1
+	);
+	await commerceThemeMiniumCatalogPage.configurationIFrameSaveButton.click();
+	await commerceThemeMiniumCatalogPage.configurationIFrameCloseButton.click();
+	await page.reload();
+
+	expect(
+		await commerceThemeMiniumCatalogPage.orderByButton.innerText()
+	).toContain(sortingOption1);
+	expect(
+		await commerceThemeMiniumCatalogPage.firstCardItem.innerText()
+	).toContain(firstCardItem);
+
+	await commerceThemeMiniumCatalogPage.orderByButton.click();
+	await commerceThemeMiniumCatalogPage.selectSorting(sortingOption2);
+
+	expect(
+		await commerceThemeMiniumCatalogPage.firstCardItem.innerText()
+	).toContain(firstCardItemAfterChange);
+}
+
+test('LPD-18714 Setting default sort for commerce products', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	commerceThemeMiniumCatalogPage,
+	page,
+}) => {
+	const siteName1 = 'Minium' + getRandomInt();
+
+	const site1 = await apiHelpers.headlessSite.createSite({
+		name: siteName1,
+		templateKey: 'minium-initializer',
+		templateType: 'site-initializer',
+	});
+
+	apiHelpers.data.push({id: site1.id, type: 'site'});
+
+	const siteName2 = 'Minium' + getRandomInt();
+
+	const site2 = await apiHelpers.headlessSite.createSite({
+		name: siteName2,
+		templateKey: 'minium-initializer',
+		templateType: 'site-initializer',
+	});
+
+	apiHelpers.data.push({id: site2.id, type: 'site'});
+
+	const sortingOption1 = 'Name Ascending';
+	const sortingOption2 = 'Name Descending';
+	const sortingOption3 = 'Price High to Low';
+	const sortingOption4 = 'Price Low to High';
+	const firstCardItemSortingOption1 = 'ABS Sensor';
+	const firstCardItemSortingOption2 = 'Wheel Seal - Front';
+	const firstCardItemSortingOption3 = 'Cams';
+	const firstCardItemSortingOption4 = 'Mount';
+
+	try {
+		await setAndCheckSorting({
+			applicationsMenuPage,
+			commerceThemeMiniumCatalogPage,
+			firstCardItem: firstCardItemSortingOption1,
+			firstCardItemAfterChange: firstCardItemSortingOption3,
+			page,
+			siteName: siteName1,
+			sortingOption1,
+			sortingOption2: sortingOption3,
+		});
+
+		await setAndCheckSorting({
+			applicationsMenuPage,
+			commerceThemeMiniumCatalogPage,
+			firstCardItem: firstCardItemSortingOption2,
+			firstCardItemAfterChange: firstCardItemSortingOption4,
+			page,
+			siteName: siteName2,
+			sortingOption1: sortingOption2,
+			sortingOption2: sortingOption4,
+		});
+
+		await applicationsMenuPage.goToSite(siteName1);
+
+		expect(
+			await commerceThemeMiniumCatalogPage.orderByButton.innerText()
+		).toContain(sortingOption1);
+		expect(
+			await commerceThemeMiniumCatalogPage.firstCardItem.innerText()
+		).toContain(firstCardItemSortingOption1);
+	}
+	finally {
+		const channels =
+			await apiHelpers.headlessCommerceAdminChannel.getChannelsPage(
+				'Minium Portal'
+			);
+
+		apiHelpers.data.push({id: channels.items[0].id, type: 'channel'});
```

Comment:
can you check if the site2 creates a new channel and in case please remove also it

Reply:
I mean, please verify that everything is deleted from site2